src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -852,13 +897,19 @@
   (let [v4-catalog-handler (v4-catalog-handler
                              jruby-service
                              wrap-with-jruby-queue-limit
-                             current-code-id-fn)]
+                             current-code-id-fn)
+        compile-handler' (compile-handler
```

Comment:
So I notice that we put the transports endpoint under v3 and this one under v4. Is there any policy we want to follow with choosing one of those going forward?

Reply:
Sorry, missed this earlier.

I put them under versions that were similar to other work that we were doing, but that's not really a reason for _why_ they should be there. I could see this being a `v3` endpoint.

Reply:
We're already not doing regular API versioning (where the whole API is supposed to move to the new version when a change happens like the new `catalog` endpoint), so I guess it doesn't matter that much to anyone except us.

I think that given we just put the other net-new endpoint under v3, maybe this should go there too. It seems a little easier to remember that you only have to use `v4` for the one that has another meaning in `v3`.

Reply:
moved this to the v3 api